### ISSUE TITLE
Don't provide too much info

### ISSUE THREAD

**Issue by oliverjam** [fac19/week6-Tiger-Kingdom#51]
https://github.com/fac19/week6-Tiger-Kingdom/blob/218115b3f2ef0cdf87c3aa62656232fd44cda92d/src/handlers/loginPost.js#L38-L52

Usually it would be a good thing that you're providing the user actionable error messages. However when it comes to password security it's usually not a good idea to be too specific about what went wrong. Otherwise you're giving a potential hacker more info: e.g. "User not found" means they've not got a valid email, whereas "Password does not match" tells them they're trying a valid account and can keep testing passwords.

**Comment by Roger-Heathcote** [comment]
Yep I see that. That was a much to aid in our debugging as anything.
